tests/rotatingMHD/diffusion_fixed_step_size.output

```diff
@@ -0,0 +1,14 @@
+
+cycle time_step cells dofs         L2               H1             Linfty      
+    0    0.5000 16384 16641 3.328e-02      - 2.958e-01      - 6.659e-02      - 
+    1    0.2500 16384 16641 1.861e-04 -14.96 1.654e-03 -14.97 3.723e-04 -14.97 
+    2    0.1250 16384 16641 5.654e-05  -3.44 5.027e-04  -3.44 1.131e-04  -3.44 
+    3    0.0625 16384 16641 1.176e-05  -4.53 1.068e-04  -4.47 2.356e-05  -4.53 
+    4    0.0312 16384 16641 2.690e-06  -4.26 3.325e-05  -3.37 5.430e-06  -4.23 
+
+cycle time_step cells dofs        L2              H1            Linfty      
+    0    0.5000 16384 16641 8.982e-03     - 7.982e-02     - 1.797e-02     - 
+    1    0.2500 16384 16641 9.461e-03  0.15 8.408e-02  0.15 1.893e-02  0.15 
```

Comment:
Is something wrong here?

Reply:
Not only that but the magnitude of the convergence rates are off. A second order scheme is used, so second order or lower would be expected.